docs/decisions/010-deprecations.md

```diff
@@ -0,0 +1,59 @@
+# Deprecations
+
+Date: 2024-11-18
+
+Status: proposed
+
+## Context
+
+Starting the development of our design system from scratch, we have been using version 0 for a long time.
+As [SEMVER suggests][semver-zero-version], we want to communicate to our users that the design system is at an early stage and any feature is not considered stable.
+However, from the time our users started to use the design system and test it, they wanted the components they would not change under the hands.
```

Comment:
```suggestion
However, from the time our users started to use the design system and test it, they wanted the components which would not change under the hands.
```
?

Reply:
This was suggested by Grammarly. But I checked this somewhere else and `that` is even better. Thanks for the check.